docs/03-tutorials/00-api-exposure/apix-10-expose-and-secure-workload-with-certificate.md

```diff
@@ -2,13 +2,15 @@
 title: Expose and secure a workload with a certificate
 ---
 
-This tutorial shows how to expose and secure a workload with mutual authentication using Kyma's mutual TLS Gateway.
+This tutorial shows how to expose and secure a workload with mutual authentication using mutual TLS Gateway.
 
 ## Prerequisites
 
 This tutorial is based on a sample HttpBin service deployment and a sample Function. To deploy or create one of those, follow the [Create a workload](./apix-01-create-workload.md) tutorial.
 
-Before you start, Set up [mTLS Gateway](../00-security/sec-03-setup-mtls-gateway.md) to allow mutual authentication in Kyma and make sure that you exported the [bundle certificates](../00-security/sec-03-setup-mtls-gateway#steps).
+Before you start, Set up:
+- [Custom Domain](./apix-02-setup-custom-domain-for-workload.md) - skip step 5 (Create a Gateway CR)
```

Comment:
if readers hop over to the Custom Domain document, will they remember to skip step 5? I would probably forget and blindly perform all steps. What happens if they perform step 5?

Consider adding a hint to that procedure that step 5 doesn't apply in certain (specified) cases.

Reply:
Note added.